pydy/system.py

```diff
@@ -429,9 +417,8 @@ def _args_for_gen_ode_func(self):
 
         """
 
-        args = (self.eom_method.forcing_full,
-                self.coordinates,
-                self.speeds,
+        args = (self.force,
```

Comment:
I agree force isn't a good name. I guess technically, it's an impulse (as in, change of generalized mass * generalized position derivative?). But since no one uses that terminology it's probably not helpful!

Matlab just calls it "odefun." We could call it RHS or function or f. Any thoughts?